ci-operator/step-registry/ipi/install/vsphere/registry/ipi-install-vsphere-registry-commands.sh

```diff
@@ -22,7 +22,7 @@ spec:
       storage: 10Gi
 EOF
   oc create -f "${SHARED_DIR}/registry-pvc.yaml" -n openshift-image-registry &&
-  oc patch config.imageregistry.operator.openshift.io/cluster --type=merge -p '{"spec":{"managementState":"Managed","rolloutStrategy":"Recreate","replicas":1,"storage":{"managementState":"Managed","pvc":{"claim":"registry-pvc"}}}}'
+  oc patch config.imageregistry.operator.openshift.io/cluster --type=merge -p '{"spec":{"managementState":"Managed","rolloutStrategy":"Recreate","replicas":2,"storage":{"managementState":"Managed","pvc":{"claim":"registry-pvc"}}}}'
```

Comment:
I think we should use the default `rolloutStrategy` here (which is `RollingUpdate`). `Recreate` will cause some disruptions.
WDYT?